modules/programs/vscode.nix

```
@@ -56,12 +61,28 @@ in {
         '';
       };
 
+      disableUpdateCheck = mkOption {
+        type = types.bool;
+        default = false;
```

```suggestion
        default = true;
```
I mean this should default to true

---

As commented [here](https://github.com/nix-community/home-manager/pull/2965#discussion_r886076681), this would generate a config and subsequently a conflict with an existing `User/settings.json` for people that don't manage their vscode config via HM.